src/action_creators.js

```diff
@@ -57,6 +57,17 @@ export function setReadyForBill(orderId) {
   };
 }
 
+export function closeOrder(orderId, cashAmount, creditAmount, creditTipAmount) {
```

Comment:
Would it be more clear if we passed in an amount object, i.e. { cashAmount, creditAmount, creditTipAmount }